simple_email_confirmation/models.py

```diff
@@ -126,7 +127,7 @@ def add_email_if_not_exists(self, email):
         """
         try:
             address = self.email_address_set.get(email=email)
-        except EmailAddress.DoesNotExist:
+        except get_email_address_model().DoesNotExist:
```

Comment:
Maybe investigate setting `EmailAddressModel = get_email_address_model()` above this class so that the function is only called once. This pattern is used in: https://github.com/django/django/blob/master/django/contrib/auth/views.py#L28

Reply:
I was having trouble calling the method at the module level because the method would be called before the app loads all the models. I will chase it down a little further.

Reply:
Django calls `get_user_model()` at import time with the use of a `require_ready` kwarg in the method call to `apps.get_model`. This kwarg was implemented in Django 1.11 with [this commit](https://github.com/django/django/commit/625cd5bcb315154ed28061fefaf639aff54da7c2#diff-e0a696d19bf764562a439d3080a30fda). Then used in `get_user_model` shortly after that in [this commit](https://github.com/django/django/commit/cb7bbf97a74fa7800865e3615f196ad65dc4f281#diff-82ec6fd74a1cab41408fbf5bf0998f58). Looks like both were part of the Django 1.11 release. 

In order to call `get_email_address_model` at the module level with 1.11 would take some work, but to call it with versions < 1.11 is not likely worth the effort. I definitely wouldn't include either in this PR. 😄

Reply:
Sounds good, thanks for looking into it.